packages/inbound-mail-parser/package.json

```diff
@@ -27,7 +27,7 @@
   },
   "dependencies": {
     "@sendgrid/helpers": "^7.6.2",
-    "mailparser": "^2.3.4"
+    "mailparser": "^3.3.0"
```

Comment:
Is there a reason we can't upgrade to [v3.4.0](https://github.com/nodemailer/mailparser/releases/tag/v3.4.0)?

Reply:
Sorry yeah, this is still a draft PR. I was just testing to fix the build failures. I will update the packages to the latest :)

Reply:
Done.